go.mod

```diff
@@ -6,7 +6,6 @@ require (
 	github.com/bradleyfalzon/ghinstallation/v2 v2.1.0
```

Comment:
The changes done on `go.mod` and `go.sum` files are not manually done by you, right?
Very briefly, I'm just curious about what is removed and why and how are they removed. Can you please give some idea?

Reply:
it actually failed in CI and told me to run `go mod tidy`

Reply:
makes sense, thanks!